server.js

```diff
@@ -0,0 +1,132 @@
+const path = require('path')
+const fs = require('fs').promises
+
+// Import/export paths
+const srcIcons = path.join(__dirname, '/src/Icon')
+const destIcons = path.join(__dirname, '/build/lib')
+
+// File format
+const ext = '.svg'
+
+// SVGO settings
+const SVGO = require('svgo')
+const svgo = new SVGO({
+  multipass: true,
+  plugins: [
+    {
+      convertPathData: {
+        floatPrecision: 2,
+        transformPrecision: 4
+      }
+    },
+    {
+      cleanupNumericValues: {
+        floatPrecision: 2
+      }
+    },
+    {
+      collapseGroups: true
+    },
+    {
+      removeTitle: true,
+    },
+    {
+      removeViewBox: false,
+    },
+    {
+      removeUselessStrokeAndFill: true,
+    },
+    {
+      removeAttrs: {
+        attrs: ['xmlns', 'fill-rule', 'clip-rule']
+      }
+    },
+    {
+      mergePaths: {
+        force: true
+      }
+    }
+  ]
+});
+
+(async () => {
+  // Read the source directory of SVGs
+  let icons = await fs.readdir(srcIcons)
+
+  // We only want .svg, ignore the rest
+  icons = icons.filter(i => path.extname(i).toLowerCase() === ext)
+
+  // Remove .svg from name
+  icons = icons.map(i => path.basename(i, ext))
+
+  // Sort alphabetically
+  icons = icons.sort()
+
+  // Array of promises which do the fetching of the files
+  let processed = icons.map(i => fs.readFile(path.resolve(srcIcons, i + ext), 'utf8'))
+  processed = await Promise.all(processed)
+
+  // Optimise them with SVGO
+  processed = processed.map(i => svgo.optimize(i))
+  processed = await Promise.all(processed)
+
+  // Get the data from the SVGO object
+  processed = processed.map(i => i.data)
+
+  // Do our custom tweaks to the output
+  processed = processed.map((i, idx) =>
+    i
+      .replace('<svg', `<svg aria-hidden="true" class="svg-icon icon${icons[idx]}"`)
```

Comment:
Is this needed because svgo strips the aria/class from the svgs?

Reply:
These aren't present in the source files since our design tools offer us no way to add these to its output. We process the optimized Figma-output here instead.

Reply:
Ah, makes sense! Not a blocker, just wondering. Might be worth it to stick a little comment in there explaining

Reply:
Commented the heck out of this